.github/workflows/ci.yaml

```diff
@@ -215,3 +215,14 @@ jobs:
 
       - name: Dependency Review
         uses: actions/dependency-review-action@6c5ccdad469c9f8a2996bfecaec55a631a347034 # v3.1.0
+
+  verify:
```

Comment:
As I said somewhere before, this should probably go into its own PR checks flow.

https://github.com/spf13/viper/blob/master/.github/workflows/checks.yaml